e2e/tests/storage_rate_limit_test.go

```diff
@@ -404,3 +482,36 @@ func (s *StorageTestSuite) TestQueryBucketRateLimit() {
 	s.Require().Equal(queryBucketRateLimitResponse.IsSet, true)
 	s.Require().Equal(queryBucketRateLimitResponse.FlowRateLimit, sdkmath.NewInt(100000000000000))
 }
+
+func (s *StorageTestSuite) TestSetBucketFlowRateLimit_Discontinue() {
+	sp, user, bucketName, _, _, _ := s.createObject()
```

Comment:
Need to create & seal object on a `New` gvg, otherwise it will not trigger the previous case.